src/pybind/mgr/cephadm/services/cephadmservice.py

```diff
@@ -551,6 +551,18 @@ def pre_remove(self, daemon: DaemonDescription) -> None:
 class MgrService(CephService):
     TYPE = 'mgr'
 
+    def allow_colo(self) -> bool:
+        if self.mgr.get_ceph_option('mgr_standby_modules'):
+            # traditional mgr mode: standby daemons' modules listen on
+            # ports and redirect to the primary.  we must not schedule
+            # multiple mgrs on the same host or else ports will
+            # conflict.
+            return False
```

Comment:
This leaved a gap:

```
ceph config set mgr mgr_standby_modules true
ceph orch apply mgr count-per-host: 2
sleep 120
ceph config set mgr mgr_standby_modules false
ceph orch restart mgr 
```
And then have some fun debugging port conflicts.

Reply:
Hmm, yes.  I think if a standby mgr sees the option go false then it should respawn.